examples/python/trigger-pagerduty-incident/handler/handler.py

```diff
@@ -0,0 +1,143 @@
+import sys, json, os
+import urllib3
+import requests
+import json
+
+# Surpress SSL warnings
```

Comment:
Can we make this optional, e.g. via environment var, please? We should not default to insecure in our functions (older ones likely need cleanup as well...)